### ISSUE TITLE
Backup: File size (2198076580) is greater than 2 GiB

### ISSUE THREAD

**Issue by abdulrahman1s** [Dokploy/dokploy#275]
### To Reproduce

1. Set up a PostgreSQL database with data exceeding 2 GiB.
2. Configure the backup functionality to use Cloudflare R2 as the provider.
3. Initiate the backup process.
4. Observe the error message indicating the file size exceeds 2 GiB.

### Current vs. Expected behavior

The PostgreSQL database backup functionality fails when dealing with large databases. Specifically, an error message is displayed stating: "File size (2198076580) is greater than 2 GiB."

### Provide environment information

```bash
OS: Ubuntu 24.04 LTS x86_6 
Dokploy version: v0.4.0
VPS Provider: OVH
```

### Which area(s) are affected? (Select all that apply)

Databases

**Comment by Siumauricio** [comment]
Thank you for reporting this issue, I will try to fix this weekend!

**Comment by saidul** [comment]
This could be related to either reading the whole file into memory or using PutObjectCommand in https://github.com/Dokploy/dokploy/blob/e7ed3c300bd253770134f367b8a43cb73b7cf449/apps/dokploy/server/utils/backups/utils.ts#L30-L37

Using multipart upload may solve this issue. 

Also using rclone (discussed in #416 ) will also solve this issue.

**Comment by Siumauricio** [comment]
Closed in #469
